lib/phut.rb

```diff
@@ -1,5 +1,5 @@
-require 'phut/cli'
 require 'phut/open_vswitch'
 require 'phut/parser'
 require 'phut/setting'
 require 'phut/version'
+require 'phut/vhost_daemon'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.